src/main.cpp

```diff
@@ -1,16 +1,86 @@
 #include <Arduino.h>
-#include "IOManagement.h"
-#include "const.h"
+
 #include "canPDC.h"
+#include "const.h"
+#include "IOManagement.h"
+#include "motor_control.h"
+#include "adc.h"
+
+//For random
+#include <stdlib.h>
+#include <time.h>
 
+int counter;
+
+//Counter for debug prints 20 ticks equates to a second
+int counterExpiry = 20;
 CANPDC canBus(CAN1, DEF);
 
+
+#if DEBUG_TECHNIQUE == 1
+void initData() {
```

Comment:
I think `randomizeData()` better describes what this function does. Usually `init()` functions are only run once.